<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HippSlicer</class>
 <widget class="qMRMLWidget" name="HippSlicer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>359</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="verticalLayout">
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BIDS directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="HippUnfoldDirSelector">
        <property name="HippUnfoldDir" stdset="0">
         <string>Input path to Hippunfold results</string>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkDirectoryButton" name="OutputDirSelector">
        <property name="OutputDir" stdset="0">
         <string>Path to output directory</string>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Config:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkPathLineEdit" name="configFileSelector">
        <property name="vCastSender" stdset="0">
         <string>Input vCastSender path</string>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="subj"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Files</string>
     </property>
     <layout class="QFormLayout" name="filesLayout">
      <item row="0" column="1">
       <widget class="QTableWidget" name="tableFiles">
        <property name="minimumSize">
         <size>
          <width>8</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Convert</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Visible</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
